### ISSUE TITLE
Uncaught TypeError: Cannot read property 'replace' of null

### ISSUE THREAD

**Issue by pchomik** [baldurh/atom-status-bar-blame#18]
[Enter steps to reproduce:]

1. Set cursor in place where git blame information appears
2. Click bar to see more information

**Atom**: 1.24.0 x64
**Electron**: 1.6.16
**OS**: Ubuntu 16.04.3
**Thrown From**: [status-bar-blame](https://github.com/baldurh/atom-status-bar-blame) package 2.0.2

### Stack Trace

Uncaught TypeError: Cannot read property 'replace' of null

```
At /home/user/.atom/packages/status-bar-blame/lib/utils/get-commit-link.js:29

TypeError: Cannot read property 'replace' of null
    at getCommitLink (/packages/status-bar-blame/lib/utils/get-commit-link.js:29:29)
    at EditorHandler.openCommitInBrowser (/packages/status-bar-blame/lib/editor-handler.js:207:18)
    at HTMLElement.onLinkClicked (/packages/status-bar-blame/lib/blame-status-bar-view.js:46:26)
```

### Commands

```
```

### Non-Core Packages

```
atom-ctags 5.1.2 
atom-typescript 12.3.3 
autocomplete-bash-builtins 0.3.5 
autocomplete-python 1.10.5 
busy-signal 1.4.3 
git-control 0.9.0 
hyperclick 0.1.5 
intentions 1.1.5 
last-cursor-position 0.9.2 
linter 2.2.0 
linter-python 3.1.2 
linter-ui-default 1.7.1 
platformio-ide-terminal 2.8.0 
project-manager 3.3.5 
status-bar-blame 2.0.2 
```

**Comment by baldurh** [comment]
Hi! Thanks for the bug report. Is it possible you do not have a remote named `origin` in your repository?

**Comment by pchomik** [comment]
Yes. This project doesn't have origin.

**Comment by ghost** [comment]
@pchomik @baldurh Same error too. May be prompt for remote selection if no origin is found.

**Comment by baldurh** [comment]
Fixed by d3417b986f5f9b12a6a5ffd337a2ef985009c1cc and released as 2.0.3
Thanks for your help 😃 
@zenati please create a new issue if you wish feature to be added. For now this just displays a tooltip notifying you there’s no origin remote.
